mitmproxy/tools/console/flowdetailview.py

```diff
@@ -33,6 +33,10 @@ def flowdetails(state, flow: mitmproxy.flow.Flow):
         req = None
         resp = None
     metadata = flow.metadata
+    comment = flow.comment
+
+    if len(comment) > 0:
```

Comment:
Nit: `if flow.comment:` is a bit more Pythonic, `bool("")` evaluates to False; every other `str` to True.